### PR TITLE
feat(molecule/avatar): export molecule avatar badge consts

### PR DESCRIPTION
**Pull request by javiauso**
## molecule/avatar
**TASK**: None

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
Export AtomAvatarBadge consts

### CODE REVIEW

**Review by midudev — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Review by Sabri1209 — APPROVED** [review]

**Review by andresz1 — APPROVED** [review]

**Review by andresin87 — CHANGES_REQUESTED** [review]
wait

File: components/molecule/avatar/src/index.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ import AtomImage from '@s-ui/react-atom-image'
...
 import useConvertStringToHex from './useConvertStringToHex'
 import AvatarFallback from './AvatarFallback'
-import AvatarBadge from './AvatarBadge'
+import AvatarBadge, {
+  AVATAR_BADGE_STATUSES,
+  AVATAR_BADGE_PLACEMENTS,
+  AVATAR_BADGE_SIZES
+} from './AvatarBadge'
 
 export const AVATAR_SIZES = {
```
Comment (andresin87) [inline components/molecule/avatar/src/index.js:14]: rename it to `MOLECULE_AVATAR_STATUSES` plzz

**Review by andresin87 — CHANGES_REQUESTED** [review]

File: components/molecule/avatar/src/index.js (hunk 1 above)
Comment (andresin87) [reply components/molecule/avatar/src/index.js:14]: preserve the exporting naming 4 retrocompatibility

Comment (javiauso) [reply components/molecule/avatar/src/index.js:14]: Ok, I will export like `MOLECULE_AVATAR_SIZES`

Comment (javiauso) [reply components/molecule/avatar/src/index.js:14]: @andresin87 https://github.com/SUI-Components/sui-components/pull/1359/commits/dfb8ad8e8bace0fbb4e9dd0f3c809e5248ecad9f

File: components/molecule/avatar/src/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import AvatarBadge, {
   AVATAR_BADGE_SIZES
 } from './AvatarBadge'
 
-export const AVATAR_SIZES = {
+const AVATAR_SIZES = {
```
Comment (andresin87) [inline components/molecule/avatar/src/index.js]: export too, 4 retrocompatibility

**Review by andresin87 — CHANGES_REQUESTED** [review]

File: components/molecule/avatar/src/index.js (hunk 2 above)
Comment (javiauso) [reply components/molecule/avatar/src/index.js:14]: Yes, is exported at bottom: https://github.com/SUI-Components/sui-components/pull/1359/files#diff-de7b3615c3bc225b5005541eaf5929362ec3e7c6d21d9fd139842817d0e56bf5R91

Comment (javiauso) [reply components/molecule/avatar/src/index.js]: Done @andresin87 https://github.com/SUI-Components/sui-components/pull/1359/commits/32d6c888be04d9377d8a57cfa081cbdaec4f94b5

**Review by andresin87 — APPROVED** [review]

**Pull request merged**